<launch>
    <arg name="turn_active" default="false" />
    <node pkg="kinematic_path_tracking" type="pure_pursuit.py" name="pure_pursuit" output="screen" respawn="true">

        <param name="rate" value="50" />

        <param name="base_link_frame" value="ugr/car_base_link" />

        <remap from="/input/path" to="/pathplanning/path" />
        <remap from="/input/odom" to="/ugr/car/odometry/filtered/odom"/>
        <remap from="/output/drive_velocity_controller/command" to="/ugr/car/drive_velocity_controller/command" />
        <remap from="/output/steering_position_controller/command" to="/ugr/car/steering_position_controller/command" />
        <remap from="/output/target_point" to="/pure_pursuit/target_point"/>
    </node>
</launch>